src/services/HttpClientService.js

```diff
@@ -1,15 +1,12 @@
 import axios from "@/axios.config";
-import { showNotification } from "@/utils/notification.js";
 
 async function fetchData(endpoint) {
     try {
         const response = await axios.get(endpoint);
         if (response.status === 200) {
-            showNotification("Successfully retrieved!", "success");
             return response.data;
         }
     } catch (error) {
-        console.error("Fetching data failed:", error);
         throw error;
```

Comment:
In stead of throwing an error, we should show a notification showing the message "Failed to fetch data from the server".